code/modules/mechs/mech_misc.dm

```diff
@@ -27,153 +27,153 @@
-//						cell.charge = cell.maxcharge * P
-
-//				P = rand(80,100) / 100 //Deplete tank
-//				if(internal_tank)//remove 50-100% of airtank contents
-//					internal_tank.air_contents.remove(internal_tank.air_contents.total_moles * P)
-
-//				lose_equipment(50)//Lose modules
-//				random_internal_damage(15)//Internal damage
-
-//				P = rand(5,50) / 100 //Set hull integrity
-//				health = initial(health)*P
-//				misconfigure_systems(15)
-
-
-//			if (41 to 80)
-//			//Decomissioned
-//			//The exosuit is a writeoff, it was tossed into storage for later scrapping.
-//			//Wasnt considered worth repairing, but you still can
-//			//Power cell missing, internal tank completely drained or ruptured/
-//			//65% chance for each type of internal damage
-//			//90% chance to lose each equipment
-//			//System settings will be randomly configured
-//				var/P
-//				if (prob(15))
-//					cell.rigged = 1//Powercell will explode if you use it
-//				else if (prob(50))//Remove cell
-//					QDEL_NULL(cell)
-
-//				if (cell)
-//					P = rand(0,20) / 100 //or deplete it
-//					cell.charge =  cell.maxcharge * P
-
-//				lose_equipment(90)//Lose modules
-//				random_internal_damage(50)//Internal damage
-
-//				if (!hasInternalDamage(MECHA_INT_TANK_BREACH))//If the tank isn't breaches
-//					qdel(internal_tank)//Then delete it
-//					internal_tank = null
-
-//				P = rand(5,50)/ 100 //Set hull integrity
-//				health = initial(health)*P
-//				misconfigure_systems(45)
-
-
-//			if (81 to 100)
-//			//Salvage
-//			//The exosuit is wrecked. Spawns a wreckage object instead of a suit
-//				//Set the noexplode var so it doesn't explode, then just qdel it
-//				//The destroy proc handles wreckage generation
-//				noexplode = 1
-//				qdel(src)
-
-//		//Finally, so that the exosuit seems like it's been in storage for a while
-//		//We will take any malfunctions to their logical conclusion, and set the error states high
-
-//		//If the tank has a breach, then there will be no air left
-//		if (hasInternalDamage(MECHA_INT_TANK_BREACH) && internal_tank)
-//			internal_tank.air_contents.remove(internal_tank.air_contents.total_moles)
-
-//		//If there's an electrical fault, the cell will be complerely drained
-//		if (hasInternalDamage(MECHA_INT_SHORT_CIRCUIT) && cell)
-//			cell.charge = 0
-
-
-//		//Code for interacting with damage+power warnings, an unported aurora feature
-//		/*
-//		process_warnings()//Trigger them first, if they'll happen
-
-//		if (power_alert_status)
-//			last_power_warning = -99999999
-//			//Make it go into infrequent warning state instantly
-//			power_warning_delay = 99999999
-//			//and set the delay between warnings to a functionally infinite value
-//			//so that it will shut up
-
-//		if (damage_alert_status)
-//			last_damage_warning = -99999999
-//			damage_warning_delay = 99999999
-
-//		process_warnings()
-//		*/
+	matter = list(MATERIAL_STEEL = 7, MATERIAL_GLASS = 3, MATERIAL_ALUMINIUM = 5)
```

Comment:
This whole commented section is not needed and should be removed. We don't keep chunks of commented code by contribution guides.

Reply:
We may still want oldification code for when oldification will be readded on mechs.

Reply:
It will not work for mob based mechs anyway, and could be looked up on git when need arises.